pages/common/git-stripspace.md

```diff
@@ -0,0 +1,16 @@
+# git stripspace
+
+> Read text, such as commit messages, notes, tags and branch descriptions, from the standard input and clean it in the manner used by Git.
```

Comment:
Perhaps this will read better?
```suggestion
> Read text (e.g. commit messages, notes, tags, and branch descriptions) from the standard input and clean it into the manner used by Git.
```

Reply:
Sure.  I copied it straight from https://git-scm.com/docs/git-stripspace.